ruby/ql/lib/codeql/ruby/security/InsecureRandomnessQuery.qll

```diff
@@ -0,0 +1,35 @@
+/**
+ * Provides default sources, sinks and sanitizers for detecting Insecure Randomness
+ * vulnerabilities, as well as extension points for adding your own.
+ */
+
+private import codeql.ruby.DataFlow
+private import codeql.ruby.TaintTracking
+import InsecureRandomnessCustomizations::InsecureRandomness
+
+/**
+ * A taint-tracking configuration for detecting Insecure Randomness vulnerabilities.
+ * DEPRECATED: Use `InsecureRandomnessFlow`
+ */
+deprecated class Configuration extends TaintTracking::Configuration {
```

Comment:
I don't think there's any need to add a deprecated class. We should remove this.